File: Scripts/laptop/regular-expressions/go/regular-expressions.go
package main
import "regexp"

func executeTask(i int) int {
  str := ""
  str = "this is a matching string"
  matched, _ := regexp.MatchString(".*string$", str)
  if matched { }

  // Substitute
  pattern := regexp.MustCompile("matching")
  result := pattern.ReplaceAllString(str, "istyfied")
  _=result
	return i + 1
}

func main() {

   r := int(1)
  // Test
  for i := 0; i < 10000000; i++ {
	r = executeTask(i + r)
  }

}

File: Scripts/laptop/regular-expressions/r/regular-expressions.r

executeTask <- function(i) {
	
pattern <- "string"
text1 <- "this is a matching string"
grep(pattern, c(text1))  # 1
gsub(pattern, "istyfied", c(text1))
	return(i+1)
}

r = 0

for(i in 0:10000000) {
	r = executeTask(i)
}
